src/Agent.Listener/Configuration/ConfigurationManager.cs

```diff
@@ -365,7 +365,28 @@ public AgentSettings LoadSettings()
                 {
                     AgentSettings settings = _store.GetSettings();
                     var credentialManager = HostContext.GetService<ICredentialManager>();
-                    VssCredentials creds = credentialManager.LoadCredentials();
+
+                    // Get the credentials
+                    VssCredentials creds;
+                    while (true)
```

Comment:
We don't persist credentials other than OAuth token that does not have permission to unregister. I'll take out the while loop.